compiler/ztests/dynamic-field-cut.yaml

```diff
@@ -0,0 +1,41 @@
+script: |
```

Comment:
Nit: There are some cut and put operator tests in compiler/op/ztests so move this and dynamic-field-put.yaml there and name them {cut,put}-dynamic-field.yaml in keeping with the existing tests.

Reply:
Fair

Reply:
Indeed, I do.